assets/sass/05-blocks/utilities/_editor.scss

```diff
@@ -179,3 +179,16 @@
 	margin: 0;
 	margin-left: var(--global--spacing-horizontal);
 }
+
+@media only screen and (min-width: 482px) {
+
+	.wp-block[data-align="left"] > * {
+		max-width: 290px;
```

Comment:
Can we use a variable here, rather than hardcoding this width?

Reply:
I tried the same same calc we use on the front-end but there were lots of issues with it. Hardcoding a value seemed like a reasonable compromise.
We don't have any related vars that I could see, and adding a new one just for the purpose of using it here didn't seem right

Reply:
Maybe we use something based on the [sass variable](https://github.com/WordPress/twentytwentyone/blob/07eab795ee6c80c662293d800081af22c39facac/assets/sass/03-generic/breakpoints.scss#L9) then? Something like `#{0.5 * $default_width}`.